system-int-test/test/cmr/system_int_test/bootstrap/bulk_index/concepts_test.clj

```diff
@@ -37,6 +37,30 @@
         exp-items (set (map #(dissoc % :status) expected))]
     (is (= exp-items search-items))))
 
+(defn- create-read-update-token
+  "Create a token with read/update permission."
+  []
+  (let [admin-read-update-group-concept-id (e/get-or-create-group (s/context) "admin-read-update-group")]
+    (e/grant-group-admin (s/context) admin-read-update-group-concept-id :read :update)
+    ;; Create and return token
+    (e/login (s/context) "admin-read-update" [admin-read-update-group-concept-id])))
+
+(deftest index-system-concepts-test-with-update-token
+  (s/only-with-real-database
+   ;; Disable message publishing so items are not indexed as part of the initial save.
+   (core/disable-automatic-indexing)
+
+   ;; Remove fixture ACLs
+   (let [response (ac/search-for-acls (u/conn-context) {} {:token (tc/echo-system-token)})
```

Comment:
Does this test work if you don't use the echo-system-token? It should work with any token that has the System Ingest Permission.

Reply:
I tried using the token it created, it worked too. Also, looks like I don't even need to remove the fixture ACLs for this test to work.  I just copied it from the other tests.